modules/ROOT/assets/attachments/configuration-properties-legacy.yaml

```diff
@@ -7,9 +7,12 @@ properties:
     type: boolean
     default: 'true'
     description: |+
-      Use `admin_interface_authentication=false` to turn of the secure remote administration feature introduced in 3.0.
+      This is set to `true` by default and requires https://docs.couchbase.com/sync-gateway/current/rest-api-access.html#lbl-secure-users[Couchbase Server RBAC-users] to authenticate Admin REST API access and send requests to the Admin API.
```

Comment:
Just a small edit suggestion here: 
Admin REST API -> https://docs.couchbase.com/sync-gateway/current/rest-api-admin.html[Admin REST API]

(creating a link to that documentation as well)

Reply:
Done!